client/src/helpers/qnAHelper.js

```diff
@@ -251,11 +251,11 @@ module.exports = {
   },
 
   qListScrollClass (count) {
-    let newClass;
+    let show;
     if (count > 1) {
-      newClass = 'questionList scroll container'
+      show = true
```

Comment:
Is this new method for showing modals/ extra questions? Wondering why the transition from changing class to changing properties. The latter is what I do, wondering if it is easier than having to style CSS for multiple things?